pkg/util/docker/networkstats.go

```diff
@@ -15,24 +15,48 @@ import (
 	log "github.com/cihub/seelog"
 )
 
-// NetworkStat stores network statistics about a Docker container.
-type NetworkStat struct {
+// InterfaceNetStats stores network statistics about a Docker network interface
+type InterfaceNetStats struct {
+	NetworkName string
 	BytesSent   uint64
 	BytesRcvd   uint64
 	PacketsSent uint64
 	PacketsRcvd uint64
 }
 
-func collectNetworkStats(containerID string, pid int, networks []dockerNetwork) (*NetworkStat, error) {
+// ContainerNetStats stores network statistics about a Docker container per interface
+type ContainerNetStats struct {
+	Stats []*InterfaceNetStats
```

Comment:
Will there ever be any other fields on this struct? If no, we could just do `type ContainerNetStats []*InterfaceNetStats`. Not a big deal either way though.

Reply:
Probably none indeed.
I started with `type ContainerNetStats []*InterfaceNetStats`, but could not range over it, got a tip for that?

Reply:
Should be able to range over it like any list I think. Example: https://play.golang.org/p/fPLHMDtEsA